src/core_modules/capture-core/components/WidgetEventEdit/DataEntry/epics/editEventDataEntry.epics.js

```diff
@@ -47,44 +50,51 @@ export const openEditEventInDataEntryEpic = (action$: InputObservable, store: Re
             return batchActions(openEventForEditInDataEntry(eventContainer, orgUnit, foundation, program, state.enrollmentDomain.enrollment?.events));
         }));
 
-
 const runRulesForEditSingleEvent = (store: ReduxStore, dataEntryId: string, itemId: string, uid: string, fieldData?: ?FieldData) => {
     const state = store.value;
     const formId = getDataEntryKey(dataEntryId, itemId);
     const eventId = state.dataEntries[dataEntryId].eventId;
-    const event = state.events[eventId];
-    const { programId } = state.currentSelections;
+    const event = state.events[eventId];  // TODO: Refactor
```

Comment:
There are a couple of `TODOs` in this file. I feel that it would be helpful if there is more context given in the comments or if a Jira ticket number is mentioned. Thanks!

Reply:
Improved the comments. Thanks for the input.